equinor/EchoComponents
# feat: #48658 - Add react datepicker wrapped component

# Description

Takes the react datepicker component from the Echopedia Web and creates a component here. Ditto team is working on creating it through EDS with timepicker. 

But we could use this one to speed up removing the modules from Echopedia Web that have dependencies to components such as this. So this is a preferable first step before taking the EDS component. This is styled to replace the existing one, not updated much. 

# Screenshots

**Closed**
![image](https://user-images.githubusercontent.com/25321939/133251179-b56864cf-083c-426f-8ad8-979f65280b5e.png)

**Open**
![image](https://user-images.githubusercontent.com/25321939/133251248-340c11b0-8fc4-43fd-939b-0400bf8ae8d8.png)

# How to test

See if it fits the common use cases for date picker currently used in Echopedia Web
